src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Common/Content/footer.html.twig

```diff
@@ -0,0 +1,50 @@
+<footer class="footer footer-transparent d-print-none">
+    <div class="container-xl">
+        <div class="row text-center align-items-center">
+            <div class="col-12 col-lg-12">
+                <ul class="list-inline list-inline-dots">
+                    <li class="list-inline-item">
+                        Copyright © 2024
+                        <a href="https://sylius.com/" target="_blank" class="link-secondary">Sylius</a>.
+                        All rights reserved.
+                    </li>
+                </ul>
+            </div>
+
+            <div class="col-lg-12 ms-lg-auto">
+                <ul class="list-inline list-inline-dots m-1">
+                    <li class="list-inline-item">
+                        <a href="https://docs.sylius.com/en/1.13" target="_blank" class="link-secondary" rel="noopener">
+                            Stable Documentation
```

Comment:
I wasn't sure if we should also include translations for these new texts, so I decided to leave it for comments.